### PR TITLE
Update shortwave aerosol lookup tables.

### PR DESCRIPTION
**Pull request by sriharshakandala**
## Purpose 
Update shortwave aerosol lookup tables.

Updated shortwave aerosol lookup tables are temporarily added directly to the repository. This file is expected to be available directly from the artifact source repository (`https://github.com/earth-system-radiation/rrtmgp-data`) in their upcoming release.

----
- [x] I have read and checked the items on the review checklist.

### CODE REVIEW

**Review by szy21 — APPROVED** [review]

File: src/ArtifactPaths.jl [hunk 1]
```diff
@@ -23,11 +23,15 @@ function get_lookup_filename(optics_type::Symbol, λ::Symbol)
     @assert optics_type ∈ (:gas, :cloud, :aerosol)
     @assert λ ∈ (:lw, :sw)
     basedir = get_artifact_path()
+    currdir = Base.@__DIR__
```
Comment (Sbozzolo) [inline src/ArtifactPaths.jl:26]:
```suggestion
    currdir =  @__DIR__
```

Base is not needed, in case you wanted to remove the qualification

**Review by Sbozzolo — APPROVED** [review]
Even if it is a temporary solution, it would be good to document this new file, mentioning how it differs from the published one and how we got it. We can remove the note once the upstream data is updated

File: src/ArtifactPaths.jl [hunk 2]
```diff
@@ -23,11 +23,15 @@ function get_lookup_filename(optics_type::Symbol, λ::Symbol)
...
 
     return optics_type == :gas ? joinpath(basedir, "rrtmgp-gas-" * (λ == :lw ? "lw-g256.nc" : "sw-g224.nc")) :
            (
         optics_type == :cloud ? joinpath(basedir, "rrtmgp-clouds-" * (λ == :lw ? "lw.nc" : "sw.nc")) :
-        joinpath(basedir, "rrtmgp-aerosols-merra-" * (λ == :lw ? "lw.nc" : "sw.nc"))
+        (
+            λ == :lw ? joinpath(basedir, "rrtmgp-aerosols-merra-lw.nc") :
+            joinpath(currdir, "../lookup_tables/aerosol/rrtmgp-sw-inputs-aerosol-optics.nc")
+        )
```
Comment (charleskawczynski) [inline src/ArtifactPaths.jl:34]: Could we change this to a more flattened pattern? For example:

```julia
    nc_file = Dict()
    nc_file[:gas, :lw] = joinpath(basedir, "rrtmgp-gas-lw-g256.nc")
    nc_file[:gas, :sw] = joinpath(basedir, "rrtmgp-gas-sw-g224.nc")
    nc_file[:cloud, :lw] = joinpath(basedir, "rrtmgp-clouds-lw.nc")
    nc_file[:cloud, :sw] = joinpath(basedir, "rrtmgp-clouds-sw.nc")

    nc_file[:merra, :lw] = joinpath(basedir, "rrtmgp-aerosols-merra-lw.nc")
    nc_file[:merra, :sw] = joinpath(currdir, "../lookup_tables/aerosol/rrtmgp-sw-inputs-aerosol-optics.nc")
    return nc_file[optics_type, λ]
```
? Or something?

Comment (charleskawczynski) [reply src/ArtifactPaths.jl:34]: I get that what currently exists is fewer instructions, but it's a lot less easy to read/extend IMO.

**Comment by sriharshakandala** [comment]
@szy21 : Please confirm if the updated documentation citing the source for this new file is accurate!

**Comment by szy21** [comment]
> @szy21 : Please confirm if the updated documentation citing the source for this new file is accurate!

Yes, it's good.

File: src/ArtifactPaths.jl (hunk 2 above)
Comment (sriharshakandala) [reply src/ArtifactPaths.jl]: A dictionary would be a bit round-about in this case. However, I updated the code to enhance readability. This is a bit more inefficient but should be as readable as the dictionary version!

**Pull request merged**